ner4opt/constants.py

```diff
@@ -0,0 +1,107 @@
+class Constants:
+    """Class for all constants."""
+
+    MODELS_DIRECTORY = 'models'
+
+    LEXICAL = 'lexical'
+    LEXICAL_PLUS = 'lexical_plus'
+    SEMANTIC = 'semantic'
+    HYBRID = 'hybrid'
+
+    # CRF Models
+    LEXICAL_MODEL_NAME = 'lexical.pkl'
+    LEXICAL_PLUS_MODEL_NAME = 'lexical_plus.pkl'
+    HYBRID_MODEL_NAME = 'hybrid.pkl'
+
+    # Deep Models
+    # Roberta Model
+    SEMANTIC_DEEP_MODEL = 'skadio/ner4opt-roberta-v1'
```

Comment:
To be consistent with the above, why not call it SEMANTIC_MODEL_v1_NAME and SEMANTIC_MODEL_v2_NAME

Reply:
OR, even more explicitly.. SEMANTIC_ROBERTA_MODEL

Reply:
btw, can we please be explicit about which roberta models are these? Please add their original roberta name/link in the comments here

Reply:
Yes, I added more details to make things clear about these roberta models